gating/check/run

```diff
@@ -293,7 +299,7 @@ if [ "${TEST_RPC_MAAS}" != "False" ]; then
     bash maas-ceph-rpco.sh
   fi
 
-  if [ "${RE_JOB_SCENARIO}" == "functional" ]; then
+  if [ "${RE_JOB_SCENARIO}" =~ ^functional ]; then
```

Comment:
Whats the logic here?

Reply:
Execute this code block if RE_JOB_SCENARIO starts with functional.  This will match functional, functional_luminous, functional_mimic, etc.